src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -131,7 +133,7 @@
     <Compile Include="Sha512Tests.cs" />
     <Compile Include="TripleDesProvider.cs" />
     <Compile Include="TripleDesTests.cs" />
-    <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
+    <Compile Condition="'$(TargetsUnix)' == 'true'" Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
```

Comment:
I feel like we normally have our conditions on ItemGroups instead of Compile line-items, can we keep that trend?